test/metabase/lib/convert_test.cljc

```diff
@@ -179,8 +183,6 @@
 
     [:value nil {:base_type :type/Number}]
 
-    [:aggregation 0 {:display-name "Bean Count"}]
```

Comment:
Since we need a source uuid in pMBQL, a standalone aggregation conversion is no longer valid.